myscripts.js

```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector('.openNav');
+const closeBtn = document.querySelector('.menu-list');
+const contactLink = document.querySelector('.contact-link');
+
+menuBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '100%';
+  document.getElementById('myLinks').style.display = 'block';
+});
+
+closeBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '0%';
+});
+
+portfolioLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
+aboutLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
+contactLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
```

Comment:
To make sure your code is easy to read and maintain, it would be great if you could use the DRY method. This means "Don't Repeat Yourself" and encourages you to avoid duplicating code. By doing this, you can reduce the risk of errors and make it easier to update your code in the future. 😊